### PR TITLE
Student answer

### PR DESCRIPTION
**Pull request by franzmoro**
<s>please review after #274 </s>

related #261 

## Tasks
  - [x] separate query for questions/answers/student answer/ correct answer
  - [x] flag correct answer in green (keep same if student has responded correctly
  - [x] flag wrong answer in red
  - [x] tests

### CODE REVIEW

**Comment by franzmoro** [comment]
perhaps best to look at it after #274 gets merged? your choice @sohilpandya @shouston3

**Comment by shouston3** [comment]
@sohilpandya and I will review after #277 is merged

File: server/lib/getQuizDetailsStudent.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+var query = require('./query');
+
+/**
+ * Returns quiz information from the database.
```
Comment (shouston3) [inline server/lib/getQuizDetailsStudent.js]: Can you update the jsdoc description

File: src/js/components/review/quiz-question.js [hunk 2]
```diff
@@ -2,27 +2,29 @@ import React, { PropTypes } from 'react';
...
     const showButton = !isAnswerShowing && !is_lecturer;
-    const showCorrectAnswer = is_lecturer || isAnswerShowing;
+    const showAnswers = is_lecturer || isAnswerShowing;
+    const answerIsWrong = showAnswers && correct_answer !== response;
 
     let aClasses = classnames("column answer box", {
-        "correct_answer": showCorrectAnswer &&
-            'a' === question.correct_answer.toLowerCase()
+        "correct_answer": showAnswers && 'a' === correct_answer.toLowerCase(),
+        "wrong_answer": response && answerIsWrong && 'a' === response.toLowerCase()
```
Comment (shouston3) [inline src/js/components/review/quiz-question.js]: Question, does the wrong answer show for the lecturer as on line 9 we're not checking for `!is_lecturer`

Comment (franzmoro) [inline src/js/components/review/quiz-question.js]: might not be so clear, but the lecturer query does not have a ```response``` field in the question. do you think it's best (for readability) to just check for ```!is_lecturer```?

File: server/lib/getQuizDetailsStudent.js (hunk 1 above)
Comment (franzmoro) [inline server/lib/getQuizDetailsStudent.js]: addressed

**Comment by franzmoro** [comment]
@shouston3 comments addressed

**Comment by iteles** [comment]
@Danwhy I don't see any updates to the readme here. If you feel some are warranted, please assign to @shouston3  :blush:

**Pull request merged**